README.md

```diff
@@ -12,26 +12,28 @@ npm install road2crypto-wallet-address-validator
 
 ## Usage
 
-Import the `checkCryptoAddress` function and use it to validate wallet addresses.
+Import the `isWalletValid` function and use it to validate wallet addresses.
 
 ```typescript
-import checkCryptoAddress from "road2crypto-wallet-address-validator";
-import { ValidationError } from "road2crypto-wallet-address-validator/dist/types";
+import isWalletValid from "road2crypto-wallet-address-validator";
 
-const result = checkCryptoAddress("your_wallet_address_here");
+const result = isWalletValid("your_wallet_address_here");
 
-if (result.error) {
-  console.error("Invalid address:", result.error);
+if (!result.valid) {
+  console.error("Error message:", result.error?.message);
```

Comment:
I'm not sure we should be logging anything here, the error should be handled on the client/backend that uses this util

Reply:
Sam for other cases

Reply:
This is README file

Reply:
Oh I see, I thought it was in the code